airflow/providers/databricks/operators/databricks.py

```diff
@@ -123,6 +124,30 @@ def _handle_databricks_operator_execution(operator, hook, log, context) -> None:
     log.info("View run status, Spark UI, and logs at %s", run_page_url)
 
 
+def update_job_for_repair(operator, hook, job_id, run_state) -> None:
```

Comment:
would be great if we can improve type annotaiton

Reply:
working on it